README.md

```diff
@@ -77,6 +77,23 @@ To display the content of the rich editor field you must call the twig filter:
 
 You can see an example in the [test application](/tests/Application/templates/bundles/SyliusShopBundle/Product/Show/Tabs/Details/_description.html.twig)
 
+### Filter the elements
+
+If you want to filter the elements which are available for your field, you can use the `tags` option when you build your form.  
+As example:
+
+```php
+$builder->add('description', RichEditorType::class, [
+    'required' => false,
+    'label' => 'sylius.form.product.description',
+    'tags' => ['product'],
+]);
+```
+
```

Comment:
Maybe we can add also the YAML example ? (I cannot add it in suggestions due to the YAML block)

In order to add tags to your Ui Element, you have to add it inside the YAML declaration :  
```yaml
monsieurbiz_sylius_richeditor:
    ui_elements:
        app.google_maps:
            # ... 
            tags: ["product"]
```